package.json

```diff
@@ -25,7 +25,9 @@
     "rebuild": "npm i && npm run clean && npm run build:library && npm run pack:library && npm run build && npm run build:demo",
     "test": "node ./scripts/test.js",
     "test:Chrome": "node ./scripts/test.js -- --browsers Chrome",
-    "publish": "node ./scripts/publish.js"
+    "publish": "node ./scripts/publish.js",
+    "lint:errors": "eslint \"./**/*.js\" --quiet",
+    "lint:all": "eslint \"./**/*.js\""
```

Comment:
Can we change it to:
"eslint './**/*.js' --quiet"

Reply:
No we can't  because eslint could not match the patters when it is with single quotes 
![image](https://user-images.githubusercontent.com/29014573/157828076-abce49c5-b3d1-47c3-ac2a-8916597a5940.png)